packages/server/controllers/guilds.js

```diff
@@ -14,7 +14,29 @@ async function getGuild(guildId) {
   return query;
 }
 
+async function getGuildMembers(guildId) {
+  const getMembers = await prisma.guildMembers.findMany({
+    where: {
+      guildId: guildId,
+    },
+    select: {
+      members: {
+        select: {
+          userId: true,
+          firstName: true,
+          lastName: true,
+        },
+      },
+    },
```

Comment:
We also need to get the avatar icon for each guild member to display in the list.